src/server/debug.rs

```diff
@@ -1275,15 +1297,23 @@ mod tests {
         // region 2 with peers at stores 21, 22, 23.
         init_region_state(engine, 2, &[21, 22, 23]);
 
-        assert!(debugger.remove_failed_stores(vec![13, 14, 23]).is_ok());
+        debugger
+            .remove_failed_stores(vec![13, 14, 21, 23], Some(vec![1]))
```

Comment:
How to know whether `Some(vec![1])` work?

Reply:
21 and 23 won't be removed from region 2. I have add some comments for that. PTAL.